.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+
+### Contributing to the model
+
+Want to contribute to the model with some additions or improvements? Consider starting by raising an issue and assign it to yourself to describe what you want to achieve. This way, we reduce the risk of duplicated efforts and you may also get suggestions on how to best proceed, e.g. there may be half-finished work in some branch that you could start with. Also, feel free to browse our [open issues](https://github.com/SysBioChalmers/yeast-GEM/issues) and our [ongoing projects](https://github.com/SysBioChalmers/yeast-GEM/projects): Anything tagged with "help wanted" is open to whoever wants to implement it!
+
+Here's how to set up yeast-GEM for local development to contribute smaller features or changes that you can implement yourself:
+
+1. First of all, make sure that you have all [requirements](https://github.com/SysBioChalmers/yeast-GEM#required-software---contributor) for contributing to yeast-GEM. Note that COBRA and RAVEN should be cloned in your computer and not just downloaded.
+
+2. Fork the yeast-GEM repository on GitHub (go to https://github.com/SysBioChalmers/yeast-GEM & click on the upper right corner).
+
+3. Clone your fork locally:
+    ```
+    git clone https://github.com/<your Github name>/yeast-GEM.git
+    ```
+	
+4. Check out the branch that you want to contribute to. Most likely that will be ``devel``:
+    ```
+    git checkout devel
+    ```
+	
+5. Create a branch for local development based on the previously checked out branch ([see below](#branching-model) for details on the branching model and how to name your branch):
+    ```
+    git checkout -b name-of-your-branch
+    ```
+	
+6. Now you can make your changes locally!
+    * Always make your changes in Matlab and never directly editing the model files. For loading the model use `loadYeastModel.m`, and for saving the model use `saveYeastModel.m`. The latter will ensure all 3 versions of the model (`.xml`, `.yml` & `.txt`) get updated in the same way.
+    * If your changes are minor (e.g. a single chemical formula you wish to correct), you can do it directly from the command line.
+    * If your changes are not so small and require several steps, create a script that loads the model, reads data (if applicable), changes the model accordingly, and saves the model back.
+    * Each script should start with a commented section describing the script, explaining the parameters, and indicating your name and the date it was written. Existing functions can clarify what style should be used.
+    * Store scripts in the appropriate folder in `/ComplementaryScripts` and data (as `.tsv` files) in the appropriate folder in `/ComplementaryData`. If you think no folder is adequate for your script/data, feel free to create your own folder. Note that binary data such as `.mat` structures or `.xls` tables cannot be stored in the repo (as they cannot be version-controlled, and they increment too much the size of the repo).
```

Comment:
You can also just add common binary file formats to the .gitignore, such as `.mat` and similar Matlab files, and `.xlsx` and other Office files. This wouldn't require a text change here though.

Reply:
Good idea, although one can always `git add --force`

Reply:
@edkerk this is actually already implemented, hence the wording "cannot"
@mihai-sysbio was not aware of `git add --force`, hopefully future PRs don't do it :)